chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -450,10 +450,6 @@ impl Client {
             .into_iter()
             .collect();
 
-        // Since chunk validators will receive the newly deployed contracts as part of the state witness (as DeployActions in receipts),
-        // they will update their contract cache while applying these deploy actions, thus we can exclude code-hash for these contracts from the message.
-        let predeployed_contract_accesses =
```

Comment:
this is not needed anymore, since we are doing the filtering in the `ContractStorage::finalize` now.